sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb

```diff
@@ -51,7 +51,7 @@
    "metadata": {},
```

Comment:
<p>I'm not sure about this. If we don't want to force the user to install the latest versions (if it's not necessary) then we should ensure that a minimum version of sagemaker/smdebug is at least installed before running the rest of the cells in this notebook. For example, we could create a <code>requirements.txt</code> in the same directory with something like:</p><p><code>smdebug &gt;= 1.0.0</code></p><p><code>sagemaker &gt;= 2.23.0</code></p><p> </p><p>And then this cell could be simplified to</p><p><code>pip install -r requirements.txt</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>The module not found issue anyways require the library updates, and in either way customers would upgrade to the latest versions. Also I would like to keep the kernel restart line to make sure the update is applied. I also wanted to avoid updating the libraries all the time, so set <code>install_needed</code>  to <code>false<span class="ql-cursor">﻿</span></code></p>

Reply:
<p>And gave customers the option to turn on the line just by simply switching <code>False</code>  to <code>True</code> .</p><p> </p>